src/cmd/manage_nsfs.js

```diff
@@ -92,7 +99,8 @@ async function main(argv = minimist(process.argv.slice(2))) {
         } else if (type === TYPES.IP_WHITELIST) {
             await whitelist_ips_management(argv);
         } else {
-            throw_cli_error(ManageCLIError.InvalidConfigType);
+            // we won't get here (we check it before)
```

Comment:
please don't keep code that is irrelevant

Reply:
But it is more readable when reading the function than removing this `else` and making the `else if` before the new `else`.
Also in case someone adds a new type it will get an error until he adds a new condition.

Reply:
Discussed and I changed the comment so it will be clear ("should" instead of "would").